tools/topology/topology2/include/common/pin_format.conf

```diff
@@ -0,0 +1,153 @@
+# Class definition for audio format for module input/output pin object
+# pin_form objects can be instantiated as:
+#
+# Object.Base.pin_format."1" {
+#	type			"input"
+#	rate			48000
+#	sample_container_size	16
+#	valid_bit_depth	16
+#	interleaving_style	"interleaved"
+# }
+#
+
+Class.Base."pin_format" {
+
+	DefineAttribute."instance" {}
+
+	DefineAttribute."index" {
```

Comment:
value of 0 as index is valid? As far as I know the pin0 format is derived from the generic base format, so index = 0 is ignored.

Reply:
thats right pin 0 will be ignored. Now changed the min to 1